### PR TITLE
Add support for PHP 8.4

### PR DESCRIPTION
**Pull request by fezfez**
- laminas/laminas-coding-standard:^3.0
- phpunit/phpunit:^10.5
- composer bump --dev-only 
- allow php 8.4

### CODE REVIEW

**Comment by fezfez** [comment]
this should be good to go (autocloser fail but ci is ok)

File: README.md [hunk 1]
```diff
@@ -41,12 +38,12 @@
...
 
 ```bash
-$ composer require laminas/laminas-text
+composer require laminas/laminas-text
 ```
 
 ## Documentation
 
-Browse the documentation online at https://docs.laminas.dev/laminas-text/
+Browse the documentation online at <https://docs.laminas.dev/laminas-text/>
```
Comment (froschdesign) [inline README.md]: This is not necessary as the rule has been disabled.

Comment (froschdesign) [reply README.md]: See: https://github.com/laminas/laminas-continuous-integration-action/blob/f474cab0f489b50c3b31d6ae04f0a5fc189c796d/setup/markdownlint/markdownlint.json#L9

Comment (fezfez) [reply README.md]: @froschdesign done

**Review by gsteel — APPROVED** [review]
LGTM 👍 Thanks @fezfez

**Pull request merged**